packages/robo/src/cli/commands/dev.ts

```diff
@@ -308,7 +308,12 @@ export async function buildAsync(command: string | null, config: Config, verbose
 
 			logger.debug(`> ${pkgManager} ${args.join(' ')}`)
 			const childProcess = spawn(pkgManager, args, {
-				env: { ...process.env, FORCE_COLOR: '1' },
+				env: {
+					// Only pass through essential vars, not the entire env
```

Comment:
Mmm, I'm not sure if this is a good idea. There are several reasons why one would want environment variables to be inherited.

### Example 1: Inline variables

Sometimes people pass environment variables inline as part of a script, either as a replacement or as an addition to `.env` loading.

```json
  "dev": "NODE_ENV=test robo dev",
  "start": "NODE_ENV=prod robo start"
```

### Example 2: Doppler

This is a service I personally use a lot in organizations. Doppler replaces the need for `.env` files. Similar to example 1, it injects variables to the process expecting it to be inherited.

```json
  "dev": "doppler run -- robo dev"
```


Passing only essential variables completely breaks the above two scenarios which are commonly used.